environment/requirements.txt

```diff
@@ -16,7 +16,7 @@ qiskit-finance==0.3.2
 scikit-learn==1.0.1
 tensorflow==2.9.1
 matplotlib==3.5.2
-pyscf==2.0.1
+--prefer-binary pyscf
```

Comment:
Can we add the version number here too? Or is that not compatible with the `--prefer-binary` flag?

Reply:
Actually, I see this is fixed in #50; which PR should I review?

Reply:
I closed the other PR, you can just check this one. I also added the pyscf version